src/System.Runtime.WindowsRuntime/src/System.Runtime.WindowsRuntime.csproj

```diff
@@ -32,6 +32,8 @@
     <ProjectReference Include="..\..\System.Runtime\src\System.Runtime.csproj"/>
```

Comment:
Can you please update these configurations so that the include Windows? For example Debug = > Windows_Debug and netcore50aot => Windows_netcore50aot? So that these configurations match the builds file.